code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -456,6 +457,7 @@ nobliumformation = 1001
 	priority = 1001 //Ensure this value is higher than nobstop
 	name = "Hyper-Noblium condensation"
 	id = "nobformation"
+	bypass_nobstop = TRUE	//Bypass nobsupress
```

Comment:
not the way to fix this

Reply:
you mean the var is unneccesary?

Reply:
i'm just making this actually works, not fixing it because the dude coded this only changed two numbers

Reply:
he probally didnt even test it

Reply:
i'd like to see your fix, bio (doesn't exist)